src/components/button/button.js

```diff
@@ -73,7 +89,22 @@ class Button extends React.Component {
      * @type {String}
      * @default medium
      */
-    size: PropTypes.string
+    size: PropTypes.string,
+
+    /**
+     * Sets a second bit of text under the main text, feinter and smaller.
```

Comment:
https://www.collinsdictionary.com/dictionary/english/feint-ruled-paper

I'll change this though I think this is less internationally understandable